packages/dapp-high-roller/src/components/app-wager/app-wager.tsx

```diff
@@ -14,13 +19,30 @@ export class AppWager {
   @State() betAmount: string = "";
   @State() myName: string = "";
 
-  handlePlay(e: Event, nodeProvider, cfjs): void {
+  async handlePlay(e: Event, nodeProvider, cfjs): Promise<void> {
```

Comment:
You can just leave the return type empty

Reply:
Yeah, I could, but I like to be explicit about it.